internal/cloudapi/v6/api.go

```diff
@@ -0,0 +1,42 @@
+package cloudapi
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	k6cloud "github.com/grafana/k6-cloud-openapi-client-go/k6"
+)
+
+// ValidateToken calls the endpoint to validate the Client's token and returns the result.
+func (c *Client) ValidateToken(stackURL string) (*k6cloud.AuthenticationResponse, error) {
```

Comment:
I find it would be valuable if we could perform some basic validation over the input this function receives. 

Although it could be argued that the API would let us know if anything is wrong, in the spirit of failling early, avoiding wasting resources, and also ensuring the best user-experience, I think we got a good shot at catching easy mistakes early here:
* We could validate that `stackURL` is not empty
* We could validate that `stackURL` is indeed a URL

That way we can exit early and make sure to generate a nice, user-friendly message early.